Models/Products.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿using BangazonAPI.Models;
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
```

Comment:
needs to be deleted

Reply:
I had to fix a small error in Products for my controller to work. Brett is working to remove the attributes in the models he created.